explorer/src/components/Staking/MyWithdrawals.tsx

```diff
@@ -0,0 +1,267 @@
+import { bigNumberToNumber, bigNumberToString } from '@/utils/number'
+import { BIGINT_ZERO, SHARES_CALCULATION_MULTIPLIER } from 'constants/general'
+import useDomains from 'hooks/useDomains'
+import useWallet from 'hooks/useWallet'
+import { FC, useMemo } from 'react'
+import { useConsensusStates } from 'states/consensus'
+import { SortedTable } from '../common/SortedTable'
+import { ActionsDropdown } from './ActionsDropdown'
+import { OperatorAction, OperatorActionType } from './ActionsModal'
+
+interface MyUnlockedWithdrawalsProps {
+  action: OperatorAction
+  handleAction: (action: OperatorAction) => void
+}
+
+export const MyUnlockedWithdrawals: FC<MyUnlockedWithdrawalsProps> = ({ action, handleAction }) => {
+  const { subspaceAccount } = useWallet()
+  const { selectedChain } = useDomains()
+  const { withdrawals } = useConsensusStates()
+  const myUnlockedWithdrawals = useMemo(
+    () =>
+      withdrawals
+        .filter(
+          (w) =>
+            w.account === subspaceAccount &&
+            w.totalWithdrawalAmount > BIGINT_ZERO &&
+            w.withdrawals.length > 0,
+        )
+        .map((w) => {
+          const totalStorageFeeRefund = w.withdrawals.reduce(
+            (acc, w) => acc + w.storageFeeRefund,
+            BIGINT_ZERO,
+          )
+          return {
+            operatorId: {
+              value: w.operatorId.toString(),
+              sortValue: w.operatorId,
+            },
+            totalWithdrawalAmount: {
+              value: `${bigNumberToString(w.totalWithdrawalAmount.toString())} ${selectedChain.token.symbol}`,
+              sortValue: w.totalWithdrawalAmount,
+            },
+            totalStorageFeeRefund: {
+              value: `${bigNumberToString(
+                totalStorageFeeRefund.toString(),
+              )} ${selectedChain.token.symbol}`,
+              sortValue: totalStorageFeeRefund,
+            },
+            unlockAtConfirmedDomainBlockNumber: {
+              value: w.withdrawalInShares.unlockAtConfirmedDomainBlockNumber.toString(),
+              sortValue: w.withdrawalInShares.unlockAtConfirmedDomainBlockNumber,
+            },
+            withdrawalsCount: {
+              value: w.withdrawals.length.toString(),
+              sortValue: w.withdrawals.length,
+            },
+            total: {
+              value: `${bigNumberToString(
+                (w.totalWithdrawalAmount + totalStorageFeeRefund).toString(),
+              )} ${selectedChain.token.symbol}`,
+              sortValue: w.totalWithdrawalAmount + totalStorageFeeRefund,
+            },
+          }
+        }),
+    [withdrawals, subspaceAccount, selectedChain.token.symbol],
+  )
+
+  const myUnlockedWithdrawalsList = useMemo(
+    () =>
+      myUnlockedWithdrawals.length > 0 && (
+        <div className='mt-5 flex flex-col gap-2'>
+          <div className='text-base font-medium text-grayDark dark:text-white'>
+            My Unlocked Withdrawals ({myUnlockedWithdrawals.length})
```

Comment:
Are these actually unlocked already, or waiting to be unlocked? The label indicates they are already unlocked but there is still an action dropdown with `Unlock...` options so I assume they are waiting to be unlocked. 
![image](https://github.com/user-attachments/assets/25a26230-bcf0-4a18-8118-eb5bf5463df0)

Reply:
Also, as you can see from the image above, you can't read the whole action.

Reply:
Label is likely misleading, it's more Ready to be unlocked 👍